duva/src/domains/caches/cache_manager.rs

```diff
@@ -74,6 +74,19 @@ impl CacheManager {
         Ok(format!("s:{}|idx:{}", value, current_idx))
     }
 
+    pub(crate) async fn route_append(
+        &self,
+        key: String,
+        value: String,
+        current_idx: u64,
+    ) -> Result<String> {
+        let Some(prev) = self.route_get(&key).await? else {
```

Comment:
Now, pre-logging validation has been lifted up so "fetching and adding back on" is not required. We can simply route it to cache actor. As Key is anyway the same.

Reply:
@Migorithm 

I thought that internal implementations of `CacheManager.route_append()` and `CacheManager.route_numeric_delta()` should be on same level.
Is that should be changed using `CacheActor` to handle `append()` and `numeric_delta()`, not `CacheManager` level?

Reply:
@DPS0340 Yes! Now delta change in and ofitself should be stored on the log and post validation is made